claim/netdata-claim.sh.in

```diff
@@ -96,13 +98,26 @@ fi
 MACHINE_GUID_FILE="@registrydir_POST@/netdata.public.unique.id"
 CLAIMING_DIR="${NETDATA_USER_CONFIG_DIR}/claim.d"
 TOKEN="unknown"
-URL_BASE="https://netdata.cloud"
+URL_BASE="https://app.netdata.cloud"
 ID="unknown"
 ROOMS=""
 HOSTNAME=$(hostname)
 CLOUD_CERTIFICATE_FILE="${CLAIMING_DIR}/cloud_fullchain.pem"
 VERBOSE=0
 INSECURE=0
+NETDATA_USER=netdata
+[ -z "$EUID" ] && EUID="$(id -u)"
+
+CONF_USER=$(grep 'run as user' "${NETDATA_USER_CONFIG_DIR}/netdata.conf" 2>/dev/null)
```

Comment:
You need to pattern match the literal tab character as well, something like:
```
"	run as user = "
```
to ignore commented out or invalid configuration.

Reply:
The tab isn't mandatory though - I could filter out lines with a `#` intead?

Reply:
It is mandatory, 99% sure.

Reply:
You're right, but we also need `=`

Reply:
Fixed.